plumbing/clock/testing.go

```diff
@@ -0,0 +1,52 @@
+package clock
+
+import (
+	"time"
+
+	"github.com/benbjohnson/clock"
```

Comment:
I noticed from looking that the issues that this isn't actively maintained and has some open issues about races (they might actually be addressed but issues are open).  Probably fine but something to keep in mind if we start seeing races in tests using it.

Reply:
Looks like the de-racing PR got closed down without merging and the author recommended this project: https://github.com/jonboulle/clockwork.

Reply:
Ohh nice find! I remember looking at that issue but didn't see the recommendation, will address and bubble up the changes to the validation branch.